rrfs-test/validated_yamls/templates/obtype_config/msonet_airTemperature_188.yaml

```diff
@@ -83,61 +88,98 @@
              is_in: 188
            action:
              name: inflate error
-             #inflation factor: 1.09757
              inflation variable:
                name: ObsFunction/ObsErrorFactorPressureCheck
                options:
                  variable: airTemperature
-                 inflation factor: 0.0
-                 #geovar_sfc_geomz: surface_geometric_height #FV3
-                 #geovar_sfc_geomz: surface_altitude #MPAS
-           defer to post: true
+                 inflation factor: 8.0
 
+         # Bounds Check
          - filter: Bounds Check
            apply at iterations: 0,1
            filter variables:
            - name: airTemperature
            minvalue: 100
            maxvalue: 400
 
+         # Create temporary ObsErrorData
+         - filter: Variable Assignment
+           assignments:
+           - name: TempObsErrorData/airTemperature
+             type: float
+             function:
+               name: ObsFunction/Arithmetic
+               options:
+                 variables:
+                 - name: ObsErrorData/airTemperature
+
+         # Set ObsError set "error parameter" if < "max value"
+         - filter: Perform Action
+           filter variables:
+           - name: airTemperature
+           action:
+             name: assign error
+             error parameter: &error_min 1.0
+           where:
+           - variable:
+               name: ObsErrorData/airTemperature
+             maxvalue: *error_min
+           - variable:
+               name: ObsErrorData/airTemperature
+             value: is_valid
+
+         # Set ObsError set "error parameter" if > "min value"
+         - filter: Perform Action
+           filter variables:
+           - name: airTemperature
+           action:
+             name: assign error
+             error parameter: &error_max 3.0
+           where:
+           - variable:
+               name: ObsErrorData/airTemperature
+             minvalue: *error_max
+           - variable:
+               name: ObsErrorData/airTemperature
+             value: is_valid
+
+         # Gross Error Check
          - filter: Background Check
            apply at iterations: 0,1
            filter variables:
            - name: airTemperature
-           absolute threshold: 5.0
            threshold: 5.0
-           where:
-           - variable: ObsType/airTemperature
-             is_in: 188
            action:
              name: reject
-
-         - filter: Domain Check
-           apply at iterations: 0,1
            where:
-             - variable:
-                 name: MetaData/timeOffset
-               minvalue: -0.10
-               maxvalue: 0.10
+           - variable: ObsType/airTemperature
+           - variable: QualityMarker/airTemperature
+             is_not_in: 3
```

Comment:
Also curious why do we not use the gross error checks for QualityMarker=3?

Reply:
You might have overlooked it but there is a separate gross error check for QualityMarker=3. That one uses cgross*0.7. So for this obtype cgross=5 and cgross*0.7=3.5

Reply:
Ahh yes, I did miss that. I was just looking at the track changes view on GitHub so it didn't show the other filter. Thanks!